[DOC-2550] - Code coverage and artifact tab plugin (#2014)
harness/developer-hub: docs/continuous-integration/use-ci/build-and-upload-artifacts/upload-artifacts-to-gcs-step-settings.md

* [Step Skip Condition settings](../../../platform/8_Pipelines/w_pipeline-steps-reference/step-skip-condition-settings.md)
* [Step Failure Strategy settings](../../../platform/8_Pipelines/w_pipeline-steps-reference/step-failure-strategy-settings.md)

## Confirm the upload

> After adding the steps and saving the pipeline, select **Run** to run the pipeline.

Reviewer: ```suggestion
After you add the steps and save the pipeline, select **Run** to run the pipeline.
```

On the [build details page](../viewing-builds.md), you can see the logs for each step as they run.

After the **Upload Artifacts to GCS** step runs, you can see the uploaded artifacts on GCS.

### View artifacts on the Artifacts tab

As an alternative to manually finding artifacts on GCS, you can use the [Artifact Metadata Publisher Drone plugin](https://github.com/drone-plugins/artifact-metadata-publisher) to publish artifacts to the [Artifacts tab](../viewing-builds.md). To do this, add a [Plugin step](../use-drone-plugins/plugin-step-settings-reference.md) after the **Upload Artifacts to GCS** step.

```mdx-code-block
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
```
```mdx-code-block
<Tabs>
  <TabItem value="Visual" label="Visual">
```

Configure the **Plugin** step settings as follows:

* **Name:** Enter a name.
* **Container Registry:** Select a Docker connector.
* **Image:** Enter `plugins/artifact-metadata-publisher`.
* **Settings:** Add the following two settings as key-value pairs.

>   * `file_urls`: A GCS URL using the **Bucket** and **Target** specified in the **Upload Artifacts to GCS** step, such as `https://storage.googleapis.com/GCS_BUCKET_NAME/TARGET_PATH.html`

Reviewer: ```suggestion
  * `file_urls`: A GCS URL using the **Bucket** and **Target** specified in the **Upload Artifacts to GCS** step, such as `https://storage.googleapis.com/GCS_BUCKET_NAME/TARGET_PATH.html`.
```

>   * `artifact_file`: `artifact.txt`

Reviewer: ```suggestion
  * `artifact_file`: `artifact.txt`.
```
